### PR TITLE
Add a bug report template for Github issues

### PR DESCRIPTION
**Pull request by pratul**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
```
Comment (saket) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Shouldn't we be assigning bugs ourselves instead of letting the author do it?

Comment (pratul) [reply .github/ISSUE_TEMPLATE/bug_report.md]: We'll recategorise them if they aren't "real" bugs.

**Comment by saket** [comment]
@pratul is there any way to test this template before it gets merged to master?

**Comment by vinaysshenoy** [comment]
Can we take a bug that we actually faced (something from Pivotal) and add it as an example here?

**Comment by pratul** [comment]
## Describe the bug
Deleted BPs are still shown in the recent patients UI

## To reproduce

### Scenario 1
1. Register a patient and record a BP. Go to home screen and verify that BP is shown.
2. Delete the BP from the patient summary and go back to home screen. The deleted BP is still displayed on the recent patients list.
Expected: No BP is displayed for the patient on the recent patients UI.

### Scenario 2
1. Register a patient and record multiple BPs. Go to home screen and verify that latest BP is shown.
2. Delete the latest BP from the patient summary and go back to home screen. The deleted BP is still displayed on the recent patients list.
Expected: The latest BP that was **NOT** deleted must be displayed.

## Expected behaviour 
A clear and concise description of what you expected to happen.

## Screenshots, if applicable
no

## Smartphone (please complete the following information):

    Device: Pixel 3
    OS: Android 9 
    App Version: 2019-03-25-3403

**Comment by vinaysshenoy** [comment]
> ## Describe the bug
> Deleted BPs are still shown in the recent patients UI
> 
> ## To reproduce
> ### Scenario 1
> 1. Register a patient and record a BP. Go to home screen and verify that BP is shown.
> 2. Delete the BP from the patient summary and go back to home screen. The deleted BP is still displayed on the recent patients list.
>    Expected: No BP is displayed for the patient on the recent patients UI.
> 
> ### Scenario 2
> 1. Register a patient and record multiple BPs. Go to home screen and verify that latest BP is shown.
> 2. Delete the latest BP from the patient summary and go back to home screen. The deleted BP is still displayed on the recent patients list.
>    Expected: The latest BP that was **NOT** deleted must be displayed.
> 
> ## Expected behaviour
> A clear and concise description of what you expected to happen.
> 
> ## Screenshots, if applicable
> no
> 
> ## Smartphone (please complete the following information):
> ```
> Device: Pixel 3
> OS: Android 9 
> App Version: 2019-03-25-3403
> ```

Looks good, for the most part. The "expected behaviour" section seems to be a part of the bug scenarios instead of in the "Expected behaviour" section.

Should we edit this, and paste this as an example bug in the issue template?

**Comment by pratul** [comment]
@vinaysshenoy I think we're good without the example as well because this is a fairly standard, traditional way of filing bugs.

**Pull request merged**